cognite_toolkit/cdf_tk/load.py

```diff
@@ -440,20 +471,21 @@ def load_resource(self, filepath: Path, dry_run: bool) -> Group:
                         values["scope"]["extractionPipelineScope"]["ids"] = [-1]
         return Group.load(raw)
 
-    def retrieve(self, ids: Sequence[int]) -> T_ResourceList:
-        remote = self.client.iam.groups.list(all=True).data
+    def retrieve(self, ids: SequenceNotStr[str]) -> GroupList:
```

Comment:
This magic type from the sdk, it's not intuitive...

Reply:
Problem is that string is a sequence, so when you say `Sequence[str]` both "a_string" and ["a_string"] will match it for `mypy`. 

And is it really that confusing? Sequence not a string?

Reply:
Ah....